openmetadata-ui/src/main/resources/ui/src/components/AppContainer/AppContainer.tsx

```diff
@@ -76,6 +76,7 @@ const AppContainer = () => {
       <Layout
         className={classNames('app-container', {
           ['extra-banner']: Boolean(bannerDetails),
+          ['reserve-right-sidebar']: Boolean(ApplicationExtras),
```

Comment:
Can you check if this changes is needed?

Reply:
I don't think. Because this class is not being used anywhere.

Reply:
we can remove this as we are `appending` this class directly on the `body` to add padding around the element where floating icons are there.
<img width="573" alt="image" src="https://github.com/user-attachments/assets/1467e686-9cf8-4f90-aa1f-59d0960e81f5">